css/main.css

```diff
@@ -51,6 +52,35 @@ header {
   color: white;
 }
 
+#mobile-nav {
+  z-index: 9999;
+  position: absolute;
+  background-color: black;
+  width: 100%;
+  height: 100vh;
```

Comment:
- [OPTIONAL] I think it would better to use `position: fixed` in order avoid scrolling when the menu is open :thinking: .